Client/BlazorRepl.Client.csproj

```diff
@@ -6,11 +6,19 @@
     <BlazorWebAssemblyEnableLinking>false</BlazorWebAssemblyEnableLinking>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
1 last thing: add this `ItemGroup` to all projects that have StyleCop analyzers package installed.

Reply:
Don't hesitate to request changes as many times as needed! 

Now i see what was meant by the first request. I was looking at the property group instead the editconfig item group.... _(playing: The Pixies - Where is my mind)_

Reply:
We are very pedantic as, I believe, you can already say. 😸 But keep in mind that this is for the best of both sides - you learn stuff and we learn stuff + keep the project "in best shape". :)

PS Great song btw!